packages/itwinui-react/src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -287,6 +287,11 @@ export const ColorBuilder = React.forwardRef((props, ref) => {
       </Box>
 
       <Slider
+        thumbProps={() => {
+          return {
+            'aria-label': `Hue slider`,
```

Comment:
You can remove the *"slider"* because "slider" is anyways announced by screen readers (at least the NVDA screen reader). So it would read out an extra "slider" ("Hue slider slider")

I also saw [this example](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-color-viewer/) on w3.org to confirm this (see the `aria-labeledby` on any one of the example color sliders).

```suggestion
            'aria-label': `Hue`,
```